src/RandomMapsChooser.as

```diff
@@ -208,6 +208,11 @@ namespace RandomMapsChooser {
             DrawTagsCheckboxes(tags);
         }
 
+        UI::Separator();
+        if (UI::CollapsingHeader("Map Blacklist")) {
+            blacklist = Text::OpenplanetFormatCodes(UI::InputText("Blacklist (mapUid1, mapUid2, ..., mapUidN)", blacklist));
```

Comment:
No need to use any formatting stuff b/c we're taking map uids. Also in general I'd separate raw text input from any formatting or things so you don't get weird loops or odd behavior.